### PR TITLE
Used index

### PR DESCRIPTION
**Pull request by Plevuus**
Added the service to set the used index (IAI or PM2.5)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,6 +102,15 @@ Set the device display light on or off
...
 | light     | `true`              | _Required_ | true or false                                |
 
+### `philips_airpurifier.set_used_index`
+
+Sets the device used index(IAI or PM2.5)
+
+| Field     | Value               | Necessity  | Description                                  |
+| --------- | ------------------- | ---------- | -------------------------------------------- |
+| entity_id | `"fan.living_room"` | _Required_ | Name(s) of the entities to set display light |
+| usedindex | `IAI`               | _Required_ | One of "IAI" or "PM2.5".                     |
```
Comment (GeorgeSG) [inline README.md]: To follow home assistant conventions, let's rename this to `used_index`

File: custom_components/philips_airpurifier/const.py [hunk 2]
```diff
@@ -24,6 +25,7 @@
 SERVICE_ATTR_CHILD_LOCK = "lock"
 SERVICE_ATTR_TIMER_HOURS = "hours"
 SERVICE_ATTR_DISPLAY_LIGHT = "light"
+SERVICE_ATTR_USED_INDEX = "usedindex"
```
Comment (GeorgeSG) [inline custom_components/philips_airpurifier/const.py]:
```suggestion
SERVICE_ATTR_USED_INDEX = "used_index"
```

File: custom_components/philips_airpurifier/services.yaml [hunk 3]
```diff
@@ -60,3 +60,12 @@ set_display_light:
       description: true or false
       example: true
 
+set_used_index:
+  description: Set the used index
+  fields:
+    entity_id:
+      description: Name(s) of the entities to set mode
+      example: "fan.living_room"
+    usedindex:
```
Comment (GeorgeSG) [inline custom_components/philips_airpurifier/services.yaml]:
```suggestion
    used_index:
```

File: README.md (hunk 1 above)
Comment (GeorgeSG) [inline README.md]:
```suggestion
| used_index | `IAI`               | _Required_ | One of "IAI" or "PM2.5".                    |
```

**Review by GeorgeSG — CHANGES_REQUESTED** [review]
Thanks for the PR! I added some comments - let's rename `usedindex` to `used_index`. Otherwise, looks good to me!

File: README.md (hunk 1 above)
Comment (Plevuus) [reply README.md]: Requested changes are done and successfully tested. One more change made to the readme.

**Review by GeorgeSG — APPROVED** [review]

**Comment by GeorgeSG** [comment]
Thanks again! Merged and released as `1.2.0`

**Comment by Plevuus** [comment]
Thank you! You do all the hard stuff. No need to thank me. Firstly, I'm using the added functionality myself. Secondly, I'm learning a lot. Thirdly I make (a tiny) contribution to 'the community' and lastly, I'm having Fun!

**Pull request merged**